README.md

```diff
@@ -10,7 +10,8 @@ Contributions are always welcome! Please take a look at the [contribution guidel
 
 ## Content
 
-- [Awesome-Avalonia](#awesome-avalonia)
+-
```

Comment:
why do you need changes here? 
Please revert it.

Reply:
Sorry, it should be a mistake.